src/index.js

```diff
@@ -1,10 +1,15 @@
+import 'babel-polyfill'
```

Comment:
Agreed, this has actually been in the lib but we should remove it.  We have lodash which covers most of what we're using polyfill for.

This will require a fair amount of changes, so we'll do this in another PR specifically for removing the polyfill.